py-polars/polars/lazyframe/frame.py

```diff
@@ -84,7 +84,7 @@
 from polars.slice import LazyPolarsSlice
 
 with contextlib.suppress(ImportError):  # Module not available when building docs
-    from polars.polars import PyLazyFrame
+    from polars.polars import PyLazyFrame, _expr_nodes, _ir_nodes
```

Comment:
@alexander-beedie 

Ruff automatically deletes these lines, which breaks the code. Any idea why and how to stop it?

Reply:
Nevermind! :)